task_13/app.js

```diff
@@ -0,0 +1,46 @@
+var Chef = function () {
+    this.dishes = ['Steak', 'Tea', 'Soup', 'Burger']
+    this.customers = 5
+}
+
+Chef.prototype.checkMenu = function() {
+    let dish = this.dishes[Math.floor(Math.random()*this.dishes.length)]
+    // console.log("I'd like to have ", dish)
+    return dish
+}
+
+Chef.prototype.customersFed = function() {
+    if (this.customers >=1){
+        console.log('Customer fed with yummy food')
+        this.customers--
+    } else if (this.customers == 0) {
+        console.log('All customers are fed. Done for the day!')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887246)
```suggestion
        console.log("All customers are fed. Done for the day!")
```